types/easeljs/easeljs-tests.ts

```diff
@@ -1,13 +1,11 @@
-
-
 var stage: any;
 var myContext2D: any;
 
 function test_simple() {
-    var canvas = <HTMLCanvasElement>document.getElementById('canvas');
+    var canvas = <HTMLCanvasElement> document.getElementById("canvas");
```

Comment:
why did you reformat all the files? I think these changes are not desired
also it makes it very difficult to review your PR